app/models/planned_disbursement.rb

```diff
@@ -17,7 +17,8 @@ class PlannedDisbursement < ApplicationRecord
     :receiving_organisation_name,
     :receiving_organisation_type,
     :financial_quarter,
-    :financial_year
+    :financial_year,
+    :providing_organisation_reference
```

Comment:
I see we validate the presence of name, and type, but we didn't validate the presence of `providing_organisation_reference` before. Out of curiosity, was there a reason for that?

Reply:
Generally speaking, the IATI reference of an organisation is optional, whilst IATI recommend providing one as it uniquely identifies an organisation, it can be hard to provide one.  Prior to this work we pre-filled in the providing org with BEIS but we thought there might be cases where it was not BEIS and the user may not be able to provide a reference in that case so it was optional.

Now we just set the value to BEIS we know we'll get the reference so we can expect every record to have one.

I feel like we are in a bit of an odd place right now, in theory we could drop all the columns where we only use the values to pusblish to IATI but the value is always the same, I think I prefer this odd place for now as we have data in these columns and I would hate to loose that, just in case.